proxy/create_container_interceptor.go

```diff
@@ -105,7 +110,17 @@ func (i *createContainerInterceptor) setWeaveWaitEntrypoint(container *docker.Co
 		}
```

Comment:
How can container.Entrypoint ever be nil?  Its zero-value is an empty list, no?  Or is == nil that same as len(...) == 0?  In which case, we already checked that.